src/documentation/pages/components/modal/examples/_scrollable_long.vue

```diff
@@ -0,0 +1,115 @@
+</button>
+
+<!-- Modal -->
+<div class="chi-backdrop -closed -mh--480">
+  <div class="chi-backdrop__wrapper">
+    <section id="modal-4" class="chi-modal" role="dialog" aria-label="Modal description" aria-modal="true">
+      <header class="chi-modal__header">
+        <h2 class="chi-modal__title">Modal Title</h2>
+        <button class="chi-button -icon -close" data-dismiss="modal" aria-label="Close">
+          <div class="chi-button__content">
+            <i class="chi-icon icon-x" aria-hidden="true"></i>
+          </div>
+        </button>
+      </header>
+      <div class="chi-modal__content">
+        <p class="-text -mt--0">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer at hendrerit lacus. Nunc ornare sollicitudin arcu vitae viverra. Nam in aliquam augue. Nunc dignissim purus sed massa consequat dictum.
+        </p>
+        <p class="-text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer at hendrerit lacus. Nunc ornare sollicitudin arcu vitae viverra. Nam in aliquam augue. Nunc dignissim purus sed massa consequat dictum.
+        </p>
+        <p class="-text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer at hendrerit lacus. Nunc ornare sollicitudin arcu vitae viverra. Nam in aliquam augue. Nunc dignissim purus sed massa consequat dictum.
+        </p>
+        <p class="-text">
+          Lorem ipsum dolor sit amet, consectetur adipiscing elit. Integer at hendrerit lacus. Nunc ornare sollicitudin arcu vitae viverra. Nam in aliquam augue. Nunc dignissim purus sed massa consequat dictum.
+        </p>
+        <p class="-text">
+          Nam volutpat tellus sapien, ac aliquam nisl mattis id. Etiam laoreet malesuada tristique. Morbi luctus facilisis laoreet. Sed pharetra diam consequat, ultrices sem facilisis, ornare urna. Suspendisse nec mollis massa.
+          Praesent ipsum purus, euismod quis mattis et, scelerisque in erat. Etiam laoreet dolor non suscipit laoreet. Sed nec mauris vitae ipsum mollis laoreet et sed nisi. In at metus eget neque consequat pellentesque. Aliquam erat volutpat.
+          Vestibulum ante ipsum primis in faucibus orci luctus et ultrices posuere cubilia Curae; Quisque egestas ultricies justo, vel scelerisque justo. Aliquam mollis diam ac ligula feugiat placerat sed iaculis urna. Donec in mi consequat,
+          cursus ante non, pulvinar eros. Phasellus viverra massa venenatis, pellentesque lectus auctor, condimentum nibh. Donec malesuada magna sed libero tincidunt, id tempus lectus mollis.
+        </p>
+      </div>
+      <footer class="chi-modal__footer">
+        <button class="chi-button" data-dismiss="modal">Cancel</button>
+        <button class="chi-button -primary">Save</button>
+      </footer>
+    </section>
+  </div>
+</div>
+
+<!-- JavaScript -->
+<script>chi.modal(document.getElementById('modal-trigger-4'));<\/script>`,
+      },
+    };
+  },
+})
+export default class ScrollLong extends Vue {}
```

Comment:
i should naming it more in agreement with the id and title, same with the file name. E.g. ScrollingLongContent